projects/dotcms-field-elements/src/components/dot-form/dot-form.tsx

```diff
@@ -39,7 +68,8 @@ export class DotFormComponent {
      */
     @Listen('valueChange')
     onValueChange(event: CustomEvent): void {
-        this.value[event.detail.name] = event.detail.value;
+        const { name, value, fieldType } = event.detail;
+        this.value[name] = fieldType ? fieldParamsConversionToBE[fieldType](value) : value;
```

Comment:
`fieldType` is a boolean?

Reply:
no, it's a optional `string` used for data conversion in `dot-form` when is needed